### DISCUSSION TITLE
How to check if drawing mode is on?

### DISCUSSION THREAD

**Discussion by royraihen** [burhanrashid52/PhotoEditor#322] [Q&A] [answered]
How to check if drawing mode is on?

**Comment by burhanrashid52** [comment, marked as answer]
This might help.
https://github.com/burhanrashid52/PhotoEditor/blob/master/photoeditor/src/main/java/ja/burhanrashid52/photoeditor/PhotoEditor.java#L381

**Comment by burhanrashid52** [comment]
Please use discussion for this. Thank you.
https://github.com/burhanrashid52/PhotoEditor/discussions?discussions_q=category%3AQ%26A
